auto-ipsec/racoon/README.md

```diff
@@ -91,14 +91,14 @@ To support revocation, you must run a CRL host/listener.  This will host a copy
  
 It will start up a web server and listen for notifications.  Be sure that the cert_crl_dist option is set to point to this server.
 
-If a node is revoked, the listener will update the CRL.  All the nodes will run their configured revocation actions:
+If an agent is revoked, the listener will update the CRL.  All the agents will run their configured revocation actions:
 
 ```
 local_action_update_crl
 local_action_deletesa
 ```
 
-These actions update the CRL by curling it from the tenant service and then search for and delete any IPsec security associations that were created using the revoked certificate serial number.  Within a second or two, all nodes in the network should stop communicating with the revoked host. 
+These actions update the CRL by curling it from the tenant service and then search for and delete any IPsec security associations that were created using the revoked certificate serial number.  Within a second or two, all agents in the network should stop communicating with the revoked host. 
```

Comment:
This should also be "machines".  The machines stop communicating with each other over IPsec, not agents.